src/lib/elements/Meter/Meter.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Meta } from "@storybook/react";
+
+import { Meter, meterColor } from ".";
+
+const meta: Meta<typeof Meter> = {
+  title: "Components/Meter",
+  component: Meter,
+  tags: ["autodocs"],
+  parameters: {
+    status: {
+      type: "legacy",
```

Comment:
What's the reasoning behind marking it as legacy? QA has passed according to the steps provided, I'm just interested in the rationale

Reply:
I don't think we'll be adding lots of them. Having this example allows us to work on and test the integration a bit sooner (before we have a refactored component in the library).